### PR TITLE
*: make it clear that proxy only supports v2 api now

### PR DESCRIPTION
**Pull request by xiang90**
@heyitsanthony @gyuho

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks.

File: Documentation/op-guide/configuration.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + env variable: ETCD_DISCOVERY_SRV
 
 ### --discovery-fallback
-+ Expected behavior ("exit" or "proxy") when discovery services fails.
++ Expected behavior ("exit" or "proxy") when discovery services fails. 'proxy' only supports v2 API.
```
Comment (heyitsanthony) [inline Documentation/op-guide/configuration.md]: s/'/" ?

File: Documentation/op-guide/configuration.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 
 ## Proxy Flags
 
-`--proxy` prefix flags configures etcd to run in [proxy mode][proxy].
+`--proxy` prefix flags configures etcd to run in [proxy mode][proxy]. 'proxy' only supports v2 API.
```
Comment (heyitsanthony) [inline Documentation/op-guide/configuration.md]: s/'/"?

File: etcdmain/help.go [hunk 3]
```diff
@@ -85,6 +86,7 @@ clustering flags:
 		auto compaction retention in hour. 0 means disable auto compaction.
 
 proxy flags:
```
Comment (heyitsanthony) [inline etcdmain/help.go]: (v2 API only)?

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**